components/VaultBox.tsx

```diff
@@ -139,6 +140,33 @@ function	VaultBox({vault, settings, noStrategies}: {vault: any, settings: TSetti
 		});
 	}
 
+	function	onTriggerModalForMetaFileMissing(): void {
+		set_fixModalData({
+			isOpen: true,
+			fix: {
+				category: 'file',
+				address: vault.address,
+				name: vault.name,
+				instructions: [
+					<span key={'step-1'}>
+						{'1. Access the vaults\' folder in the meta repo: '}
+						<a href={`https://github.com/yearn/yearn-meta/tree/master/data/vaults/${chainID}`} target={'_blank'} className={'underline'} rel={'noreferrer'}>
+							{`https://github.com/yearn/yearn-meta/tree/master/data/vaults/${chainID}`}
+						</a>
```

Comment:
Meta has been merged with yDaemon, url need some change!
```suggestion
						<a href={`https://github.com/yearn/ydaemon/tree/master/data/meta/vaults/${chainID}`} target={'_blank'} className={'underline'} rel={'noreferrer'}>
							{`https://github.com/yearn/ydaemon/tree/master/data/meta/vaults/${chainID}`}
						</a>
```